docs/private-dns/api-reference.md

```diff
@@ -0,0 +1,561 @@
+---
+title: API Reference
+sidebar_position: 3
+toc_min_heading_level: 3
+toc_max_heading_level: 4
+---
+
+<!-- 
+    The content below is simply an automatic conversion from the OpenAPI spec
+    https://api.adguard-dns.io/static/swagger/openapi.json to markdown using
+    https://swagger-markdown-ui.netlify.app/.
+
+    If you want to change it, ask the developers to change the OpenAPI spec.
+-->
+
+# AdGuard DNS API
+DNS API documentation
+
+## Version: 1.3
+
+### /oapi/v1/account/limits
+
+#### GET
+##### Summary:
```

Comment:
this subheading seems superfluous
probably, it's better to format the text in this way:

```
#### GET

 Get account limits

 ##### Responses

| Code | Description |
 | ---- | ----------- |
 | 200 | Account limits info |

#### POST

 Creates a new device

##### Responses
<...>
```

Reply:
This is auto-generated, let's keep it as is for the sake of simplicity.